src/components/Toast/Toast.tsx

```diff
@@ -68,11 +73,68 @@ const StyledIcon = styled(Icon)`
   font-size: ${getFontSize('lg')};
 `;
 
+const colorVariants = {
+  [ToastVariants.info]: ColorTypes.info500,
+  [ToastVariants.warn]: ColorTypes.warning500,
+  [ToastVariants.error]: ColorTypes.error500,
+  [ToastVariants.success]: ColorTypes.success500,
+};
+
+const iconVariants = {
+  [ToastVariants.info]: SSCIconNames.infoCircle,
+  [ToastVariants.warn]: SSCIconNames.exclTriangleSolid,
+  [ToastVariants.error]: SSCIconNames.errorSolid,
+  [ToastVariants.success]: SSCIconNames.checkCircleSolid,
+};
+
+const iconPxSizesVariants = {
+  [ToastVariants.info]: 16,
+  [ToastVariants.warn]: 14,
+  [ToastVariants.error]: 16,
+  [ToastVariants.success]: 16,
+};
+
+const ToastIconPadbox = styled(Padbox)<{ $variant?: ToastProps['variant'] }>`
+  background-color: ${({ $variant }) => getColor(colorVariants[$variant])};
+  display: flex;
+  align-items: center;
+`;
+
+const ToastIconWrapper = styled(FlexContainer)`
```

Comment:
@dclerici-ssc please use normal `div` tag instead of FlexContainer (we are trying to get rid of that thing) and set the flexbox by yourself

Reply:
I tried to get rid of this component entirely and it seems that everything works, so we probably can remove this.

Reply:
done!